src/pytorch_lightning/callbacks/progress/base.py

```diff
@@ -267,17 +267,20 @@ def get_standard_metrics(trainer: "pl.Trainer", pl_module: "pl.LightningModule")
     Return:
         Dictionary with the standard metrics to be displayed in the progress bar.
     """
+    loss_value = None
+    loss_metric = None
+
     # call .item() only once but store elements without graphs
-    running_train_loss = trainer.fit_loop.running_loss.mean()
-    avg_training_loss = None
-    if running_train_loss is not None:
-        avg_training_loss = running_train_loss.cpu().item()
+    if trainer.training:
+        loss_metric = trainer.fit_loop._results.get("training_step.train_loss")
```

Comment:
Why keep this? Getting this key is arbitrary, users could log it with `tr_loss`, or `loss`, or `perdida` :es: 

We could use the return value of the `training_step`, but I suggest removing this logic completely.

Reply:
Do you know why sometimes the value of `fit_loop._results.get("training_step.train_loss")` is of type `_ResultMetricCollection` and sometimes `_ResultCollection`?